css-positioning/relative.css

```diff
@@ -0,0 +1,41 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
+  top: 20px;
+}
+
+#charmander {
+  /* your code here */
+  position: relative;
+  top: 40px;
+}
+
+#squirtle {
+  /* your code here */
+  position: relative;
+  top: 80px;
+}
+
+#pikachu {
+  /* your code here */
+  position: relative;
+  top: 160px;
+}
+
+#jigglypuff {
+  /* your code here */
+  position: relative;
+  top: 300px;
+}
```

Comment:
Rather than applying `position: relative` to each of these, you could just add it once to the `.pokemon > img` rule, and save yourself the repetition!

Reply:
Definitely a lot of new learnings from Q&A today 😆 thank you for the feedback, Cody!!